controller/environment_blackbox_test.go

```diff
@@ -0,0 +1,65 @@
+package controller_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	testsuite "github.com/fabric8-services/fabric8-common/test/suite"
+	"github.com/fabric8-services/fabric8-env/app"
+	"github.com/fabric8-services/fabric8-env/app/test"
+	"github.com/fabric8-services/fabric8-env/configuration"
+	"github.com/fabric8-services/fabric8-env/controller"
+	"github.com/fabric8-services/fabric8-env/gormapp"
+	"github.com/fabric8-services/fabric8-env/resource"
+	"github.com/goadesign/goa"
+	"github.com/stretchr/testify/suite"
+)
+
+type EnvironmentControllerSuite struct {
+	testsuite.DBTestSuite
+}
+
+func TestEnvironmentController(t *testing.T) {
+	resource.Require(t, resource.Database)
```

Comment:
Do we want to re-use `F8_*` used in `resource.Require()` from f8-common ??
Because I see that there are few services which doesn't have `F8_*` rather AUTH service has `AUTH_*`.

Reply:
I would reuse F8_* used in resource.Require() everywhere. If we use F8_* in our make test targets then we can reuse resource.Require() from common.